spec/models/dataset_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Dataset, type: :model do
+
+  subject { FactoryGirl.build(:dataset) }
+  it 'has a valid factory' do
+    expect(FactoryGirl.create(:dataset)).to be_valid
+  end
+  it 'is invalid if the name is missing' do
+    expect(build(:dataset, {name: ''})).not_to be_valid
+  end
+
```

Comment:
Please add model tests for `created_at`, `creator_id`, `updated_at`, and `updater_id`

Also, fix spacing between lines 8&9